contracts/reverseRegistrar/L2ReverseClaimer.sol

```diff
@@ -0,0 +1,21 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.17 <0.9.0;
+
+import {ENS} from "../registry/ENS.sol";
+import {IL2ReverseRegistrar} from "../reverseRegistrar/IL2ReverseRegistrar.sol";
+
+contract L2ReverseClaimer {
+    constructor(
+        address l2ReverseRegistrarAddr,
+        ENS reverseRegistrar,
+        address claimant
+    ) {
+        IL2ReverseRegistrar reverseRegistrar = IL2ReverseRegistrar(
+            l2ReverseRegistrarAddr
+        );
+        //reverseRegistrar.setName(claimant);
```

Comment:
Should this be uncommented, or deleted?

Reply:
This contract doesn't seem to be used from any other contracts nor any tests exist. Is the contract fully built out?

Reply:
Removed for now, we can possibly add it again later